integration-tests/Cargo.toml

```
@@ -25,5 +25,5 @@ syn = "1.0"
 tempfile = "3.3"
 tokio = "1.0"
 quote = "1.0"
-workspaces = "0.7.0"
+workspaces = { version = "0.8.0", package = "near-workspaces"}
```

Let’s migrate to the new name

```suggestion
near-workspaces = { version = "0.8.0" }
```